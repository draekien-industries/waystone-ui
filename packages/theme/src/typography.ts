export const typography = {
  fonts: {
    body: '"Barlow", sans-serif',
    heading: '"Barlow", sans-serif',
    monospace: '"Fira Code", monospace',
  },
  fontSizes: {
    hero: '2.5rem',
    heading: '1.75rem',
    title: '1.5rem',
    subtitle: '1.25rem',
    body: '1rem',
    caption: '0.875rem',
    small: '0.75rem',
  },
  fontWeights: {
    thin: 300,
    regular: 400,
    medium: 500,
    semibold: 600,
    bold: 700,
  },
  lineHeights: {
    body: 1.5,
    heading: 1.125,
  },
  letterSpacings: {
    body: 'normal',
    uppercase: '0.1em',
  },
} as const;

export default typography;
