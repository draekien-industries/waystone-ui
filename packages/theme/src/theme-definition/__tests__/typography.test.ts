import { typography } from '../typography';

describe('typography', () => {
  it('should match snapshot', () => {
    expect(typography).toMatchSnapshot();
  });
});
