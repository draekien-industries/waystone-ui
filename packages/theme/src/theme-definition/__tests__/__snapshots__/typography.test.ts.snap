// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`typography should match snapshot 1`] = `
{
  "fontSizes": {
    "body": "1rem",
    "caption": "0.875rem",
    "heading": "1.75rem",
    "hero": "2.5rem",
    "small": "0.75rem",
    "subtitle": "1.25rem",
    "title": "1.5rem",
  },
  "fontWeights": {
    "bold": 700,
    "medium": 500,
    "regular": 400,
    "semibold": 600,
    "thin": 300,
  },
  "fonts": {
    "body": ""Barlow", sans-serif",
    "heading": ""Barlow", sans-serif",
    "monospace": ""Fira Code", monospace",
  },
  "letterSpacings": {
    "body": "normal",
    "uppercase": "0.1em",
  },
  "lineHeights": {
    "body": 1.5,
    "heading": 1.125,
  },
}
`;
